java/src/main/java/org/whispersystems/signalservice/api/crypto/SignalServiceCipher.java

```diff
@@ -332,58 +332,56 @@ private SignalServiceReceiptMessage createReceiptMessage(SignalServiceEnvelope e
                                                          .setType(type)
                                                          .build());
         }
+      }
```

Comment:
Github makes this look like a big change, but really I just ended this if statement here, then fixed indentation below.

Reply:
@greyson-signal To view the diff without whitespace changes, you can append `?w=1` to the URL:

https://github.com/signalapp/libsignal-service-java/pull/55/files?w=1